simple_rpc/simple_rpc.py

```diff
@@ -44,6 +44,35 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
+    def _open(self):
+        if self.is_open():
```

Comment:
Guard for double opening, should never happen.

Reply:
ok